typescript-practice/src/ts/models/user.ts

```diff
@@ -0,0 +1,18 @@
+import { createIdUser } from '../helpers/helpers';
+
+export default class User {
+  private id: number;
```

Comment:
Why each prop split by empty line?

Reply:
Because I follow this comment from eslint. 

![image](https://github.com/Nez27/typescript-training/assets/123229563/b57505d5-6d08-4847-8480-3953698d9752)